pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/internal/pex/ThinPexGenerator.java

```diff
@@ -44,7 +44,7 @@
     public ThinPexGenerator(Project project, String entryPointTemplate, Map<String, String> extraProperties) {
         this.project = project;
         this.entryPointTemplate = entryPointTemplate;
-        this.extraProperties = extraProperties == null ? new HashMap<>() : extraProperties;
+        this.extraProperties = extraProperties == null ? new HashMap<String, String>() : extraProperties;
```

Comment:
Java 7 introduced the diamond in the cases like this `HashMap<>` and will infer the types correctly.
Do we really need to repeat the types that are already declared in the field itself?